src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -98,16 +101,24 @@ def generate_default_strategy_pool():
 def generate_weighted_strategy_pool():
   """Generate a strategy pool based on probability
   distribution from multi armed bandit experimentation."""
-  query = data_types.FuzzStrategyProbability.query()
-  distribution = list(ndb_utils.get_all_from_query(query))
+  distribution = environment.get_value('STRATEGY_SELECTION_DISTRIBUTION')
 
   # If we are not able to query properly, draw randomly according to
   # probability parameters.
-  if (not distribution or
-      not environment.get_value('USE_BANDIT_STRATEGY_SELECTION')):
+  if not distribution:
+    logs.log('Cannot use weighted strategy pool. Generating default strategy pool.')
```

Comment:
Seems better to remove this before landing. Ditto for some of the other logs below.